api/agent/agent_test.go

```diff
@@ -240,23 +241,24 @@ func TestCallConfigurationModel(t *testing.T) {
 }
 
 func TestAsyncCallHeaders(t *testing.T) {
-	appName := "myapp"
+	app := &models.App{Name: "myapp"}
+	app.SetDefaults()
 	path := "/sleeper"
 	image := "fnproject/sleeper"
 	const timeout = 1
 	const idleTimeout = 20
 	const memory = 256
 	CPUs := models.MilliCPUs(200)
 	method := "GET"
-	url := "http://127.0.0.1:8080/r/" + appName + path
+	url := "http://127.0.0.1:8080/r/" + app.Name + path
 	payload := "payload"
 	typ := "async"
 	format := "http"
 	contentType := "suberb_type"
 	contentLength := strconv.FormatInt(int64(len(payload)), 10)
 	config := map[string]string{
 		"FN_FORMAT":   format,
-		"FN_APP_NAME": appName,
+		"FN_APP_NAME": app.ID,
```

Comment:
we probably want to keep name here (blech, this got hairy)

Reply:
i think that we should give an access to the app name only, will revert this one as well.